docs/next/modules/en/pages/setup/security/rbac/rbac_permissions.adoc

```diff
@@ -20,65 +20,132 @@ There are two types of permission in SUSE Observability. *System permissions* sc
- ** Power User: `everything` (all views)
- ** Guest: -
-* `unlock-node` - Unlock xref:/stackpacks/about-stackpacks.adoc#_locked_configuration_items[locked configuration items].
-* `update-permissions` - Grant/revoke permissions or change subjects.
-* `update-settings` - Update settings.
-* `update-visualization` - Change xref:/use/views/k8s-topology-perspective.adoc#_visualization_settings[visualization settings].
-* `upload-stackpacks` - Upload new (versions of) StackPacks.
-* `view-metric-bindings` - View xref:/use/metrics/k8s-add-charts.adoc[metric bindings] (via the cli)
-* `view-monitors` - View monitor configurations.
-* `view-notifications` - View notification settings.
+.Permissions
+[cols="2,2,5"]
+|===
+|Resources |Verbs |Description
+
+|agents
+|get
+|Connected agents
+
+|api-tokens
+|get
+|The API key to use for authentication with the SUSE Observability CLI.  This is unique per user.
+
+|component-actions
+|execute
+|xref:/use/views/k8s-topology-perspective.adoc#_actions[Component actions in the topology perspective].
+
+|ingestion-api-keys
+|get, create, delete
+|xref:/use/security/k8s-ingestion-api-keys.adoc[Additional API keys for data ingestion].
+
+|metric-bindings
+|get, create, update, delete
+|xref:/use/metrics/k8s-add-charts.adoc[Bindings to visualize data in the metrics perspective].
+
+|metrics
+|get^[1]^, update
+|Ingest and query metric data using PromQL
+
+|monitors
+|get, create, update, delete, execute
+|xref:/use/alerting/k8s-monitors.adoc[Determine component health with monitors].
+
+|notifications
+|get, create, update, delete
+|xref:/use/alerting/notifications/configure.adoc[Get alerted with notifications].
+
+|permissions
+|get, update
+|Manage permissions on subjects.
+
+|scoped-permissions
+|update^[1]^
+|Manage scoped permissions, granting (or revoking) access to components, metrics and traces from a single cluster.
+
+|scripts
+|execute
+|Execute a query in the SUSE Observability UI Analytics environment.
+
+|restricted-scripts
+|execute
+|Use the HTTP script and Gremlin graph API in the Analytics environment.
+
+|service-token
+|get, create, delete
+|xref:/use/security/k8s-service-tokens.adoc[Service tokens authenticate an external service].
+
+|settings
+|get, update, unlock
+|Export or import settings or unlock xref:/stackpacks/about-stackpacks.adoc#_locked_configuration_items[locked configuration items].
+
+|stackpacks
+|get, update
+|List or upload xref:/stackpacks/about-stackpacks.adoc[Stackpacks].
+
+|stackpack-configurations
+|create, update, delete
+|Manage integrations using xref:/stackpacks/about-stackpacks.adoc[Stackpacks].
+
+|sync-data
+|get, delete
+|Inspect or reset the topology and health synchronization pipelines.
+
+|system-notifications
+|get
+|Access the system notifications in the UI
+
+|topic-messages
+|get
+|Access SUSE Observability Receiver data.
+
+|topology
+|get^[1]^
+|xref:/use/views/k8s-views.adoc[Explore topology scoped views].
+
+|traces
+|get^[1]^
```

Comment:
^[1]^

Reply:
superscript for [1] - to reference the scoped permission explanation below